docs/basics/options.md

```diff
@@ -1199,31 +1280,37 @@ searchLimit: undefined, // disable search count limits
 searchLimit: 150,
 ```
 
-### `legacyLinking`
+### `flatLinking`
 
-| Config file name | CLI short form | CLI long form      | Format    | Default |
-| ---------------- | -------------- | ------------------ | --------- | ------- |
-| `legacyLinking`  |                | `--legacy-linking` | `boolean` | `false` |
+| Config file name | CLI short form | CLI long form    | Format    | Default |
+| ---------------- | -------------- | ---------------- | --------- | ------- |
+| `flatLinking`    |                | `--flat-linking` | `boolean` | `false` |
 
 :::warning NOTICE
 This feature is a v6 only feature.
 :::
 
 Set this to `true` to use the flat-folder style linking previously used in v5. This option
-will otherwise link any matches to a tracker specific folder inside of `linkDir` (if set).
+will otherwise link any matches to a tracker-specific folder inside of `linkDir` (if set).
```

Comment:
Probably don't need the new line in the middle of a sentence.

Reply:
Some of these are just to make it easier to read and edit in the IDE...